app/views/step_by_step_pages/show.html.erb

```diff
@@ -76,6 +76,16 @@
         @step_by_step_page_presenter.secondary_links_settings
       ]
     }  %>
+
+    <%= render "govuk_publishing_components/components/summary_list", {
```

Comment:
I think you can call this with just `topic_settings` in the same way that `summary_list_params` is used further up the file. 

e.g.
`<%= render "govuk_publishing_components/components/summary_list", @step_by_step_page_presenter.topic_settings  %>`